protobuf.lua

```diff
@@ -16,72 +16,107 @@ project "protobuf"
   uuid "1A92AB33-0DCB-4953-BCB9-467862B5FE1F"
 
   files {
-    prjDir .. "/src/google/protobuf/arena.cc",
-    prjDir .. "/src/google/protobuf/extension_set.cc",
-    prjDir .. "/src/google/protobuf/generated_message_table_driven_lite.cc",
-    prjDir .. "/src/google/protobuf/generated_message_util.cc",
-    prjDir .. "/src/google/protobuf/implicit_weak_message.cc",
-    prjDir .. "/src/google/protobuf/io/coded_stream.cc",
-    prjDir .. "/src/google/protobuf/io/zero_copy_stream.cc",
-    prjDir .. "/src/google/protobuf/io/zero_copy_stream_impl_lite.cc",
-    prjDir .. "/src/google/protobuf/message_lite.cc",
-    prjDir .. "/src/google/protobuf/repeated_field.cc",
-    prjDir .. "/src/google/protobuf/stubs/bytestream.cc",
-    prjDir .. "/src/google/protobuf/stubs/common.cc",
+    prjDir .. "/src/google/protobuf/any.cc",
```

Comment:
nitpick.  Can this list be sorted?  It helps with finding and adding files in the future.